src/thunderbots/software/launch/ai.launch

```diff
@@ -4,19 +4,24 @@
         Runs all of the AI nodes except for the backend (Grsim, MRF, etc.)
     -->
 
+    <param name="vision_multicast_address" value="$(optenv VISION_MULTICAST_ADDRESS 224.5.23.2)"/>
+
+    <!-- Start rosbag -->
+    <node name="rosbag_record_all" pkg="rosbag" type="record" args="-a -o $(find thunderbots)"/>
```

Comment:
Do we still want to keep rosbag recording by default?

Reply:
I don't think it makes a huge difference, but I'm inclined to keep whatever we do in master right now (so don't record).

Reply:
Removed rosbag from launchfile.